lib/cocoapods/command/spec/create.rb

```diff
@@ -60,7 +60,7 @@ def default_data_for_template(name)
           data[:author_email]  = `git config --get user.email`.strip
           data[:source_url]    = "http://EXAMPLE/#{name}.git"
           data[:ref_type]      = ':tag'
-          data[:ref]           = '0.0.1'
+          data[:ref]           = '#{s.version}'
```

Comment:
It is intentional (see last comment in PR). The non interpolated string will be used in the generated podspec.